Donation-platform/src/main/java/com/google/PersonalizedRecommendations.java

```diff
@@ -52,7 +64,8 @@
 
     // Returns mapping of charities to the scores that determine their rank on the personalzied page (higher score = better match)
     public HashMap<Charity, Double> getCharityScores(List<String> selectedTags) {
-        Collection<Charity> charities = getAllCharities();
+        Collection<Charity> charities = getAllCharities(); //DEFAULT
+        // Collection<Charity> charities = getAllHardCodedCharities(); //HARDCODED
```

Comment:
why is this hardcoded

Reply:
Because for testing purposes I still need to be able to use the hardcoded charities. I made two separate functions for getAllCharities and getAllHardCodedCharities to simplify this.

Reply:
ok.